ironic/resource_ironic_node_v1.go

```diff
@@ -585,3 +618,137 @@ func changePowerState(client *gophercloud.ServiceClient, d *schema.ResourceData,
 
 	return nil
 }
+
+// setRAIDConfig calls ironic's API to send request to change a Node's RAID config.
+func setRAIDConfig(client *gophercloud.ServiceClient, d *schema.ResourceData) (err error) {
+	var logicalDisks []nodes.LogicalDisk
+	var raid *metal3v1alpha1.RAIDConfig
+
+	raidConfig := d.Get("raid_config").(string)
+	if raidConfig == "" {
+		return nil
+	}
+
+	err = json.Unmarshal([]byte(raidConfig), &raid)
+	if err != nil {
+		return
+	}
+
+	err = checkRAIDConfigure(d.Get("raid_interface").(string), raid)
+	if err != nil {
+		return
+	}
+
+	// Build target for RAID configuration steps
+	logicalDisks, err = ironic.BuildTargetRAIDCfg(raid)
+	if len(logicalDisks) == 0 || err != nil {
+		return
+	}
+
+	// Set root volume
+	if len(d.Get("root_device").(map[string]interface{})) == 0 {
+		logicalDisks[0].IsRootVolume = new(bool)
+		*logicalDisks[0].IsRootVolume = true
+	} else {
+		log.Printf("rootDeviceHints is used, the first volume of raid will not be set to root")
+	}
+
+	// Set target for RAID configuration steps
+	return nodes.SetRAIDConfig(
+		client,
+		d.Id(),
+		nodes.RAIDConfigOpts{LogicalDisks: logicalDisks},
+	).ExtractErr()
+}
+
+// buildBIOSSettings builds bios settings for BIOS clean steps
+func buildBIOSSettings(d *schema.ResourceData, firmwareConfig *bmc.FirmwareConfig) (settings []map[string]string, err error) {
+
+	// disableCertificateVerification is default omitempty, if not specifying it as true in the install-config
+	// it would not exits in driver_info
+	var disableCertificateVerification bool
+	driverInfo := d.Get("driver_info").(map[string]interface{})
+	if _, ok := driverInfo["irmc_verify_ca"]; ok {
```

Comment:
Ugh, sorry for not spotting this earlier. We should not have any irmc-specific code here.

Reply:
Ah, sorry. It seems that we need transfer `disableCertificateVerification` from installer to here too.

Reply:
I updated the [implementation](https://github.com/openshift-metal3/terraform-provider-ironic/pull/53/commits/d8ef9d9492a198b4777c089b594edd2d04af1264#diff-373c79495b73be3d62528c3a124f7bdf8966d21f21e2f8ab49dfc1754bf4eec9R670). PTAL.

Reply:
And I also updated the [implementation](https://github.com/openshift/installer/pull/5196/commits/d924c56f59ad915ba6f88b205fcf4f08267bf956) on installer side. PTAL.